javafx/resources/css/nocfa.css

```diff
@@ -37,12 +67,13 @@
 	-fx-background-image: url(nocfa/skipb_hover.png);
 }
 
+/*
```

Comment:
move this up where it belongs and adjust it accordingly instead of commenting out please